runtime/long.go

```diff
@@ -321,6 +321,7 @@ func initLongType(dict map[string]*Object) {
 	LongType.slots.New = &newSlot{longNew}
 	LongType.slots.NonZero = longUnaryBoolOpSlot(longNonZero)
 	LongType.slots.Or = longBinaryOpSlot(longOr)
+	LongType.slots.Pow = &binaryOpSlot{longPow}
```

Comment:
I see. No I don't think you're missing anything. It's probably worth adding a comment about why we don't use longBinaryOpSlot for this method.